### PR TITLE
Fixing maximum message size

### PR DESCRIPTION
**Pull request by madhavajay**
## Description
This fixes the maximum message size.
With string encoding we were hitting issues around 256 MB, but now we can go over the rabbitmq recommended 512 MB.

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: packages/grid/backend/grid/core/celery_app.py [hunk 1]
```diff
@@ -14,3 +62,7 @@
     "grid.worker.msg_without_reply": "main-queue",
     "delivery_mode": "transient",
 }
+celery_app.worker_prefetch_multiplier = 1
+celery_app.conf.accept_content = ["application/syft"]
```
Comment (rasswanth-s) [inline packages/grid/backend/grid/core/celery_app.py:66]: MAYBE:  could you add it in the config file

Comment (rasswanth-s) [reply packages/grid/backend/grid/core/celery_app.py:66]: https://github.com/OpenMined/PySyft/blob/dev/packages/grid/backend/grid/core/celery_config.py

**Review by rasswanth-s — APPROVED** [review]
Awesome, using our custom protobuf serializer

File: tests/integration/large_message_test.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    mb = f"{mb_size} MB"
+
+    try:
+        start_time = time.time()
+        print(f"Sending {mb} sized message")
+        x_ptr = x.send(domain_client, tags=[size, mb])
+        x_ptr.block_with_timeout(120)
+        total_time = time.time() - start_time
+        print(f"Took {total_time}")
+        data_rate = mb_size / total_time
```
Comment (rasswanth-s) [inline tests/integration/large_message_test.py:37]: Amazing !!!!!!!!!!!!!!!!!! throughput calculation

Comment (rasswanth-s) [reply tests/integration/large_message_test.py:37]: was there any performance improvement madhava, using our protobuf in celery worker in communication

File: packages/grid/backend/grid/core/celery_app.py (hunk 1 above)
Comment (madhavajay) [reply packages/grid/backend/grid/core/celery_app.py:66]: yeah let me address this in a small PR so i can get this one merged and fix CI

**Pull request merged**